backend/src/main/java/com/funeat/member/domain/Member.java

```diff
@@ -67,17 +67,10 @@ public List<RecipeFavorite> getRecipeFavorites() {
     }
 
     public void modifyProfile(final String nickname, final String profileImage) {
-        if (Objects.isNull(nickname) || Objects.isNull(profileImage)) {
+        if (Objects.isNull(nickname) || Objects.isNull(profileImage) || profileImage.isBlank()) {
```

Comment:
```suggestion
        if (!StringUtils.hasText(nickname) || !StringUtils.hasText(profileImage)) {
```

`nickname`도 `MemberRequest`에서 `@NotBlank`로 하니까 둘 다 null이거나, 공백이면 에러 처리하는건 어떤가요?

Reply:
감사합니다!